### DISCUSSION TITLE
SSBH Editor BlendState error (material XML importing) - need help

### DISCUSSION THREAD

**Discussion by BeyondSmash** [ScanMountGoat/ssbh_editor#113] [Q&A] [answered]
Hi someone tried to use my tutorial for Material Animations using SSBH Editor (importing an XML file).
They received the error at the top of the picture below. I couldn't figure out what the problem was to be able to solve it. Can you help me figure out how to solve it?

![image](https://user-images.githubusercontent.com/110957189/194448958-c28ea09b-8d23-4460-8705-d43c696589a8.png)

**Comment by ScanMountGoat** [comment, marked as answer]
Make sure you have the enclosing parameter tags. It's probably easier to just copy paste the shader label into a new material in SSBH Editor or make a preset from an existing material. While it's definitely possible to make materials using MatLab or ssbh_data_json, the syntax is very finicky and can be hard to troubleshoot for people that aren't programmers.
```xml
<Parameter name="BlendState0">
...
</Parameter>
```

**Reply by BeyondSmash** [reply]
Yes I have exactly that and compared the code with vanilla smash characters regarding BlendState and it was the same.
![image](https://user-images.githubusercontent.com/110957189/194450699-826716dc-938a-463f-b79b-dd50e7c3f608.png)
I'm just trying to help someone making the animated material using SSBH Editor instead of CrossMod like you had advised in the tutorial comments section (as it would load the material in CrossMod, but not in SSBH).

Here's the link to the material code that is the reference for being imported as an XML:
https://textdoc.co/jnpJah3quRCSdixF

**Reply by ScanMountGoat** [reply]
The whole thing should be wrapped in a MaterialLibrary node like the output from MatLab. It's easier to just paste the shader label into the material editor following the steps at https://github.com/ScanMountGoat/ssbh_editor/wiki/Matl-Editor#copy-materials-by-changing-shaders.

**Reply by ScanMountGoat** [reply]
The error message will be slightly clearer on the next release. The goal of this feature is to make migration from Cross Mod easier and not to encourage people to continue using MatLab for material editing.

**Reply by BeyondSmash** [reply]
Awesome, pasting the shader worked well. Thank you for your help! Looking forward to the future updates! (Just realized I never posted this message...oops)
